build.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const fs = require('fs');
+const http = require('follow-redirects').http;
+const archiver = require('archiver');
+
+var options = {
+    host: 'uplift.shipit.staging.mozilla-releng.net',
+    path: '/coverage/supported_extensions',
+    method: 'GET',
+    headers: {"Content-Type": "text/plain"}
+}
+
+
+var req = http.request(options, (res) => {
+
+	var data = '';
+    
+    res.on('data', (chunk) => {
+        data += chunk;
+    });
+
+    res.on('end', () => {
+
+        fs.writeFile("supported_extensions.js", "const SUPPORTED_EXTENSIONS = " + data, (e) => {
```

Comment:
Can you use the nicer template strings for `const SUPPORTED_EXTENSIONS = `? https://developer.mozilla.org/docs/Web/JavaScript/Reference/template_strings